src/Persistence/Initialization/VersionSeasonSix/ChaosMixes.cs

```diff
@@ -141,62 +143,62 @@ private ItemCrafting ChaosWeaponCrafting()
         var chaosWeapon = this.Context.CreateNew<ItemCrafting>();
         chaosWeapon.Name = "Chaos Weapon";
         chaosWeapon.Number = 1;
+        chaosWeapon.ItemCraftingHandlerClassName = typeof(ChaosWeaponAndFirstWingsCrafting).FullName!;
         var chaosWeaponSettings = this.Context.CreateNew<SimpleCraftingSettings>();
-
         chaosWeapon.SimpleCraftingSettings = chaosWeaponSettings;
-        chaosWeaponSettings.MoneyPerFinalSuccessPercentage = 10000;
-        chaosWeaponSettings.SuccessPercent = 1;
+        chaosWeaponSettings.MoneyPerFinalSuccessPercentage = 10_000;
+        chaosWeaponSettings.NpcPriceDivisor = 20_000;
```

Comment:
[This](https://github.com/kyleruss/emu-server/blob/202856a74c905c203b9b2795fd161f564ca8b257/GameServer/Source/ChaosBox.cpp#L291) says 10,000; [this](https://github.com/MuSeason6FixTeam/zTeamS6.3/blob/565c579e61f150f6019dee60d22443d30ddfaf42/zGameServer/GameServer/MixSystem.cpp#L297) says 20000. But it's 20000 that gets the client displayed values.